packages/node_modules/@webex/widget-speed-dial/src/SpeedDialForm.tsx

```diff
@@ -356,8 +364,10 @@ export const SpeedDialForm = ({
                     id={name}
                     name={name}
                     defaultValue={formData.currentCallAddress}
+                    autoComplete="off"
                     label={t('form.phone.label')}
                     placeholder={t('form.phone.placeholder')}
+                    aria-label={t('form.phone.placeholder')}
```

Comment:
Could you please assign aria-label as formData.currentCallAddress  instead of aria-label={t('form.phone.placeholder')}?

Reply:
Okay